components/bulletins/BulletinCardCollection.vue

```diff
@@ -37,12 +37,12 @@ export default {
 <style lang="postcss" scoped>
 .bulletinCardCollection__cardContainer {
     @apply flex flex-wrap;
```

Comment:
**suggestion (non-blocking):** How about using grid layout here? it could help us make the margin on the left side equal to the right side. Currently, they have different widths depending on different size devices. As shown below, the right side of the margin is longer than the left side. You could refer to the code [here](https://github.com/pycontw/pycontw-frontend/blob/main/components/sponsors/SponsorCardCollection.vue#L31)